static/js/components/CandidateList.jsx

```diff
@@ -62,6 +62,13 @@ const useStyles = makeStyles((theme) => ({
   spinnerDiv: {
     paddingTop: "2rem",
   },
+  spinner: {
```

Comment:
To prevent having to copy the CSS over and over again, factor out the spinner into its own .jsx file with its own CSS.